packages/rspack/src/Compilation.ts

```diff
@@ -354,6 +363,17 @@ BREAKING CHANGE: Asset processing hooks in Compilation has been merged into a si
 		);
 	}
 
+	getCustomModule(moduleIdentifier: string) {
```

Comment:
This should be private

Reply:
This is called in Module's constructor. I change it to `__getCustomModule` to mark it as internal method.